packages/frontend/app/models/contact.ts

```diff
@@ -18,6 +18,7 @@ export enum STATUS {
 
 export default class Contact extends Identity {
   @attr() onlineStatus?: STATUS;
+  @attr('boolean', { defaultValue: false }) isPinned?: boolean;
```

Comment:
What was the reason for the attr transform? ('boolean')?

I think this is the same:
```ts
@attr() isPinned?: boolean;
```

Reply:
I agree, thought we needed a default value

Reply:
Agree, I though we needed a default value